apps/charterafrica/src/payload/blocks/Mooc.js

```diff
@@ -0,0 +1,34 @@
+import linkGroup from "../fields/linkGroup";
+
+const Mooc = {
+  slug: "mooc",
+  fields: [
+    {
+      name: "title",
+      type: "richText",
+      required: true,
+      localized: true,
+      admin: {
+        elements: ["h1"],
```

Comment:
```suggestion
        elements: [],
```

Everything will go to `h1`... shouldn't create another one.